http/headers/Alt-Svc.json

```
@@ -7,7 +7,7 @@
           "spec_url": "https://httpwg.org/specs/rfc7838.html#alt-svc",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "≤74"
```

I found the intent and bug for this:
https://groups.google.com/a/chromium.org/g/blink-dev/c/L-BRotfWD8U
https://issues.chromium.org/issues/40374026#comment61

So it looks like the right version is:

```suggestion
              "version_added": "52"
```

---

https://github.com/mdn/browser-compat-data/pull/23718